app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -90,7 +90,7 @@
 <li>disabled </li>
 </ul>
 <p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
-<p>You can <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">find out more about fees and financial support available for non-UK citizens</a>.</p>
+<p>You can <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">find out more about fees and financial support for non-UK citizens</a>.</p>
```

Comment:
This change needs moving to the maths PR (if appropriate) and the file removed from this PR